docs/3.api/1.components/13.youtube-embed.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "<YoutubeEmbed>"
+description: A simple and performant YouTube component.
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/scripts-and-assets/blob/main/modules/nuxt-third-party-capital/src/runtime/components/YoutubeEmbed.ts
+    size: xs
+---
+
+The YoutubeEmbed component is based on [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed).
+
```

Comment:
Same thing here. I think it would be nicer to have an immediate codeblock example before the params section (can be a tiny example to begin with).

Since you only have a single example below, I think it would be fine to just move it up to here.